<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blowfisher</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>71</height>
       </size>
      </property>
      <property name="title">
       <string>Тип операции:</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="encryptButton">
         <property name="text">
          <string>Шифровать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="decryptButton">
         <property name="text">
          <string>Дешифровать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="openDir">
      <property name="text">
       <string>Открыть каталог назначения в проводнике после обработки списка</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="key">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addItem">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Добавить файл</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icon/folderAdd.png</normaloff>:/icons/icon/folderAdd.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteItem">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Убрать файл из списка</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icon/delete.png</normaloff>:/icons/icon/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Обработать список</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icon/go.png</normaloff>:/icons/icon/go.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Директория назначения:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ключ:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Список обрабатываемых файлов:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QListWidget" name="listfile">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="path">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhEmailCharactersOnly</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectDir">
        <property name="toolTip">
         <string>Выбрать каталог</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutMenu"/>
   </widget>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutMenu">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>selectDir</tabstop>
  <tabstop>path</tabstop>
  <tabstop>listfile</tabstop>
  <tabstop>key</tabstop>
  <tabstop>addItem</tabstop>
  <tabstop>deleteItem</tabstop>
  <tabstop>encryptButton</tabstop>
  <tabstop>decryptButton</tabstop>
  <tabstop>start</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
